src/components/input/DatePickerWithLabelFormGroup.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { Label } from '@hospitalrun/components'
+
+interface Props {
+  name: string
+  label: string
+  value: string
+  isEditable: boolean
+  onChange: (event: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const DatePickerWithLabelFormGroup = (props: Props) => {
+  const { onChange, label, name, value, isEditable } = props
+  const id = `${name}DatePicker`
+  return (
+    <div className="form-group">
```

Comment:
Why not using our date picker component here?

Reply:
updated to use our date picker